lib/integration/referral_code.rb

```diff
@@ -0,0 +1,49 @@
+module TalonOne
+  module Integration
+    class ReferralCode
+      def initialize(raw_data)
+        @raw = raw_data
+      end
+
+      def id
+        @raw["id"]
+      end
+
+      def campaign_id
+        @raw["campaignId"]
+      end
+
+      def created
+        @raw["created"]
+      end
+
+      def advocate_profile_integration_id
+        @raw["advocateProfileIntegrationId"]
+      end
+
+      def code
+        @raw["code"]
+      end
+
+      def friend_profile_integration_id?
+        @raw["friendProfileIntegrationId"]
+      end
+
+      def start_date?
```

Comment:
this and `expiry_date?` should not have the question mark at the end. Only use a question mark for methods that return a boolean.